packages/contracts/test/plugin/plugin-setup-processor.ts

```diff
@@ -720,186 +882,164 @@ describe('Plugin Setup Processor', function () {
+        } = await installHelper(psp, targetDao2, setupC1, repoClones));
+
         let pluginUpdateParams = {
-          plugin: pluginCloneableMock.address,
-          pluginSetupRepo: ADDRESS_TWO,
-          currentPluginSetup: AddressZero,
-          newPluginSetup: AddressZero,
+          plugin: pluginCloneable,
+          pluginSetupRepo: repoClones.address,
+          currentPluginSetup: setupC1.address,
+          newPluginSetup: setupC2.address,
         };
 
-        const helpers = [AddressZero];
         await expect(
           psp.prepareUpdate(
-            targetDao.address,
+            targetDao2.address,
             pluginUpdateParams,
-            helpers,
+            helpersV1,
             EMPTY_DATA
           )
         ).to.be.revertedWith(
-          customError('PluginNonupgradeable', pluginCloneableMock.address)
+          customError('PluginNonupgradeable', pluginCloneable)
         );
       });
 
       it('reverts if `PluginSetupRepo` do not exist on `PluginRepoRegistry`', async () => {
-        const daoAddress = targetDao.address;
-        const pluginSetupV1 = pluginSetupV1Mock.address;
-
         const {plugin, helpers} = await prepareInstallation(
           psp,
-          daoAddress,
-          pluginSetupV1,
-          pluginSetupMockRepoAddress,
+          targetDao.address,
+          setupV1.address,
+          repoUUPS.address,
           EMPTY_DATA
         );
 
         const pluginSetupRepoAddr = ADDRESS_TWO;
         const pluginUpdateParams = {
           plugin: plugin,
           pluginSetupRepo: pluginSetupRepoAddr,
-          currentPluginSetup: pluginSetupV1,
-          newPluginSetup: pluginSetupV2Mock.address,
+          currentPluginSetup: setupV1.address,
+          newPluginSetup: setupV2.address,
         };
 
         await expect(
-          psp.prepareUpdate(daoAddress, pluginUpdateParams, helpers, EMPTY_DATA)
+          psp.prepareUpdate(
+            targetDao.address,
+            pluginUpdateParams,
+            helpers,
+            EMPTY_DATA
+          )
         ).to.be.revertedWith(customError('EmptyPluginRepo'));
       });
 
       it('revert if plugin is not applied', async () => {
-        const daoAddress = targetDao.address;
-        const pluginSetupV1 = pluginSetupV1Mock.address;
-
         const {plugin, helpers} = await prepareInstallation(
           psp,
-          daoAddress,
-          pluginSetupV1,
-          pluginSetupMockRepoAddress,
+          targetDao.address,
+          setupV1.address,
+          repoUUPS.address,
           EMPTY_DATA
         );
 
         const pluginUpdateParams = {
           plugin: plugin,
-          pluginSetupRepo: pluginSetupMockRepoAddress,
-          currentPluginSetup: pluginSetupV1,
-          newPluginSetup: pluginSetupV2Mock.address,
+          pluginSetupRepo: repoUUPS.address,
+          currentPluginSetup: setupV1.address,
+          newPluginSetup: setupV2.address,
         };
 
         await expect(
```

Comment:
adding this as well before calling `prepareInstallation` ensures better safety.

Reply:
What do you mean? What should I add before `prepareInstallation`?

Reply:
Closed after discussion with author